app/(technician)/farmers/page.tsx

```diff
@@ -42,11 +39,11 @@ export default async function FarmerList({ searchParams }: Props) {
           />)
         }
       </div>
-      <div className="flex justify-center gap-6 mt-6">
-        <PaginationButtons
-          prevLink={isFirst ? undefined : `?offset=${Math.max(0, +offset - PAGE_LIMIT)}`}
-          nextLink={isLast ? undefined : `?offset=${+offset + PAGE_LIMIT}`} />
-      </div>
     </main>
+    <div className="flex justify-between p-4 shadow-lg">
+      <PaginationButtons
+        prevLink={isFirst ? undefined : `?offset=${Math.max(0, +offset - PAGE_LIMIT)}`}
+        nextLink={isLast ? undefined : `?offset=${+offset + PAGE_LIMIT}`} />
```

Comment:
Just curious, what is the difference between `+offset` and `offset`?

Reply:
query params values are registered as strings, so I need to convert the `offset` string into a number by prefixing with `+`